deepchecks/nlp/utils/text_properties.py

```diff
@@ -448,6 +449,102 @@ def average_sentence_length(raw_text: Sequence[str]) -> List[float]:
     return result
 
 
+def count_unique_urls(raw_text: Sequence[str]) -> List[str]:
```

Comment:
Sorry for missing it earlier, but I actually think that for these two properties (url and email) it would be more useful to count the total number of url / emails, and not the number of unique emails/urls. If the same email appears twice within a sample, it's still a larger part of the text devoted to "emails" than if it appeared once.

If you disagree I would ask that you keep these two but add two more properties for the total number of emails & urls

Reply:
Kept the unique properties and created properties to calculate all emails and URLs.